### PR TITLE
exporting cluster-profile of SNO in case we're using a 'sno' variant

### PR DESCRIPTION
**Pull request by osherdp**

### CODE REVIEW

**Comment by osherdp** [comment]
openshift/enhancements#504
openshift/installer#4444
openshift/enhancements#543

**Review by eranco74 — COMMENTED** [review]
/lgtm

File: ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml [hunk 1]
```diff
@@ -794,6 +800,10 @@ objects:
           export OPENSHIFT_INSTALL_PRESERVE_BOOTSTRAP=true
         fi
 
+        if has_variant "sno"; then
+          export OPENSHIFT_INSTALL_CLUSTER_PROFILE_OVERRIDE="single-node-production-edge"
```
Comment (ravidbro) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml:804]: The env var name that was eventually merged is OPENSHIFT_INSTALL_EXPERIMENTAL_CLUSTER_PROFILE

**Comment by osherdp** [comment]
not needed

**Pull request closed without merging**